src/dbwrapper.h

```diff
@@ -75,6 +83,14 @@ class CDBBatch
         leveldb::Slice slValue(ssValue.data(), ssValue.size());
 
         batch.Put(slKey, slValue);
+        // LevelDB serializes writes as:
+        // - byte: header
+        // - varint: key length (1 byte up to 127B, 2 bytes up to 16383B, ...)
+        // - byte[]: key
+        // - varint: value length
+        // - byte[]: value
+        // The formula below assumes the key and value are both less than 16k.
```

Comment:
Wait, is this wrong? Looks like you're assuming both varints to be 1 byte, which the comment suggests is only up to 127B, not 16KB?